content/source/assets/javascripts/analytics.js

```diff
@@ -24,4 +24,17 @@ document.addEventListener('turbolinks:load', function() {
     params.event = 'CTA Clicked'
     return params
   })
+
+  // track navigation link clicks
+  Array.prototype.slice
+    .call(
+      document.querySelectorAll(
+        '.navbar-static-top .navigation-links a[data-track]'
+      )
+    )
+    .map((el) => {
```

Comment:
UNFORTUNATELY, ES6 lambda syntax will cause the "sprockets" build pipeline to barf. 😞 It might not explode in local dev mode, but it's caused breakages in the past for prod builds.

Reply:
Ah man I forgot about that! Good callout here, I'll revert those back to normal functions. Thank you!